components/core/src/clp/clo/OutputHandler.hpp

```diff
@@ -65,13 +107,16 @@ class ResultsCacheClient {
      * @param message The content of the log event.
      * @param timestamp The timestamp of the log event.
      */
-    void
-    add_result(std::string const& original_path, std::string const& message, epochtime_t timestamp);
+    void add_result(
```

Comment:
* Reorder methods so overrides are after non-overrides.
* We don't need the docstrings anymore.